presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeMap.java

```diff
@@ -84,4 +91,16 @@ public NodeMap(
     {
         return allNodesByHostAndPort;
     }
+
+    public NodeProvider getActiveNodeProvider(NodeSelectionHashStrategy nodeSelectionHashStrategy)
+    {
+        switch (nodeSelectionHashStrategy) {
+            case MODULAR:
+                return new ModularHashingNodeProvider(activeNodes);
+            case CONSISTENT:
```

Comment:
The class is called `HashStrategy` already. If you think it could be confusing, then to make it uniform, should we name them `MODULAR_HASHING` and `CONSISTENT_HASHING`?

Reply:
> The class is called `HashStrategy` already. If you think it could be confusing, then to make it uniform, should we name them `MODULAR_HASHING` and `CONSISTENT_HASHING`?

yeah, sounds good, that would be better